integrate ampire displays and fix g070vw01v0 timing parameters

adv-eecc opened the pull request:
fixed display timing and added support for Ampire 7" and 10.1" displays

ebenezer-samson committed:
integrate ampire displays and fix g070vw01v0 timing parameters

clayderhua commented:
Hi @adv-eecc 

About the auo_g070vw01v0 parameter in drivers/gpu/drm/panel/panel-simple.c, your "hsync" and "vsync" parameter is correct.

Follow the "AUO G070VW01 spec" page18, the minmum clock is 27M.
https://www.elecok.com/data_sheet/41386/G070VW01_V0_7.0%22_a-Si_TFT-LCD_Panel_for_AUO(EN).pdf 

I suggest the "clock =29500" in auo_g070vw01v0_mode.

static const struct drm_display_mode auo_g070vw01v0_mode = {
    .clock = 29500,
    .hdisplay = 800,
    .hsync_start = 800 + 200,
    .hsync_end = 800 + 200 + 6,
    .htotal = 800 + 200 + 6 + 50,
    .vdisplay = 480,
    .vsync_start = 480 + 20,
    .vsync_end = 480 + 20 + 4,
    .vtotal = 480 + 20 + 4 + 4,
};

Commit:
updated timings based on discussion
- g070vw01 and am800480 both run at 29.5 MHz = 60 fps
- am1280800 runs at 45 Hz only since it seems the display cable cannot handle 60 Hz at WXGA

adv-eecc commented:
@clayderhua 
I have just done another commit to adjust the timings as discussed.
